src/test/java/se/sundsvall/casedata/service/ProcessServiceTest.java

```diff
@@ -118,6 +130,25 @@ void updateProcessWhenParkingPermit(final String caseType) {
 		verifyNoInteractions(landAndExploitationIntegrationMock);
 	}
 
+	@ParameterizedTest
+	@MethodSource("paratransitTypesProvider")
+	void updateProcessWhenParatransit(final String caseType) {
+		// Arrange
+		final var errand = createErrandEntity();
+		errand.setNamespace("SBK_PARKING_PERMIT");
+		errand.setCaseType(caseType);
+
+		when(parkingPermitPropertiesMock.supportedNamespaces()).thenReturn(List.of("SBK_PARKING_PERMIT"));
+
+		// Act
+		processService.updateProcess(errand);
```

Comment:
Add test to verify behaviour when paratransitClient/paratransitIntegration throws an exception